packages/components/pie-button/src/index.ts

```diff
@@ -1,8 +1,8 @@
 import {
     LitElement, html, unsafeCSS, nothing, PropertyValues, TemplateResult,
 } from 'lit';
+import { classMap } from 'lit-html/directives/class-map.js';
```

Comment:
This is the [suggested](https://lit.dev/docs/templates/directives/#classmap) import path, I don't think we have `lit-html` listed as a dependency.
```suggestion
import { classMap } from 'lit/directives/class-map.js';
```